data/brands/amenity/bicycle_rental.json

```
@@ -1784,7 +1784,9 @@
     {
       "displayName": "Kołobrzeski Rower Miejski",
       "id": "kolobrzeskirowermiejski-4d145c",
-      "locationSet": {"include": ["pl"]},
+      "locationSet": {
+        "include": [[15.5764209, 54.1759614]]
+      },
       "tags": {
```

```suggestion
      "note": "Kołobrzeski in Poland",
      "tags": {
```

@starsep I can't make changes directly in your fork(looks like something strange), so I use review comment.

Add a note after limit entry to specific city will help other contributor know where it is. Thankyou.

---

Added in 9d0a0964e0a71ac29747076fc9b5a8ac713c5539